### PR TITLE
Add/configs/alarm configs

### PR DESCRIPTION
**Pull request by cuervodelviento**
These are the proposed schemas to interact with an alarm based on my meeting with @gmD3v  I included some small changes and applied the suggestions we talked later.

@galindocode please check (and have mercy on us 🙆‍♂️🙆‍♂️ )

### CODE REVIEW

File: doc/config-schemas/setAddAlarmUser.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+```
+
+| Campo      | Tipo   | Descripcion                   |
+| ---------- | ------ | ----------------------------- |
+| systemId   | string | Id de sistema de la particion |
+| userName   | string | Nombre del usuario            |
+| accessCode | string | codigo de acceso              |
+
```
Comment (galindocode) [inline doc/config-schemas/setAddAlarmUser.md:30]: Why not here need the `number` propiertie in the request value, for example in `setAlarmPartition` the `number` is used in the value, why not here?

File: doc/config-schemas/setAlarmUser.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        "configKey": "setAlarmUser",
+        "value":{
+            "systemId":"xxx-xxxx",
+            "userName": "Em-32",
+            "userId:"xxx-xxxx",
+            "accessCode": "5125"
+        }
+    }
+}
+
```
Comment (galindocode) [inline doc/config-schemas/setAlarmUser.md:23]: Why not here need the number propiertie in the request value, for example in setAlarmPartition the number is used in the value, why not here?

**Comment by galindocode** [comment]
In the `zones` config I see that only use zone identifier of type `int` in the `number` propertie. Could be that another manufacturer use a string identifier in `zones`?

File: doc/config-schemas/setDeleteAlarmPartitionZone.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+```json
+{
+    ...
+    "data": {
+        "deviceData": {...},
+        "configKey": "setDeleteAlarmPartitionZone",
+        "value":{
+            "number":1,
+            "systemId": "xxx-xxxx",
+            "zoneNumber":14
```
Comment (galindocode) [inline doc/config-schemas/setDeleteAlarmPartitionZone.md:18]: check this comment
https://github.com/Netsocs-Team/driver.sdk_go/pull/15#issuecomment-2031992105

**Review by galindocode — CHANGES_REQUESTED** [review]
@cuervodelviento I was merciful cr@cuervodelviento I was merciful ✝

File: doc/config-schemas/setAlarmUser.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+| Campo      | Tipo   | Descripcion                   |
+| ---------- | ------ | ----------------------------- |
+| systemId   | string | Id de sistema de la particion |
+| userId     | string | Id de sistema de el usuario   |
+| userName   | string | Nombre del usuario            |
+| accessCode | string | codigo de acceso              |
+
```
Comment (galindocode) [inline doc/config-schemas/setAlarmUser.md:32]: Why not here need the number propiertie in the request value, for example in setAlarmPartition the number is used in the value, why not here?

**Review by galindocode — CHANGES_REQUESTED** [review]

**Comment by cuervodelviento** [comment]
> @cuervodelviento, What is `FAP`?

FAP is short for Fire Alarm Panic, these are three typical physical buttons on the alarm systems, they are a way to manually trigger the alarm.

image
image

they also vary on name (Fire, Medical (assistance), Police)

image
image

**Comment by galindocode** [comment]
> > @cuervodelviento, What is `FAP`?
> 
> FAP is short for Fire Alarm Panic, these are three typical physical buttons on the alarm systems, they are a way to manually trigger the alarm.
> 
> image image
> 
> they also vary on name (Fire, Medical (assistance), Police)
> 
> image image

Excellent, I suggest add this message (with images) in the schema that contains this acronym

**Pull request merged**